app/src/store/bootstrap.ts

```diff
@@ -114,8 +114,74 @@ export const bootstrapCallInvites = createTypedAsyncThunk<
     /**
      * Handle an incoming, pending, call invite.
      */
-    const handlePendingCallInvite = async (callInvite: TwilioCallInvite) => {
-      await dispatch(receiveCallInvite(callInvite));
+    const handlePendingCallInvite = (callInvite: TwilioCallInvite) => {
+      /**
+       * Handle the call invite accepted event.
+       */
+      const handleCallInviteAccepted = (call: TwilioCall) => {
+        // dispatch the new call
+        dispatch(handleCall({ call }));
+        handleSettledCallInvite(callInvite);
+
+        const callSid = callInvite.getCallSid();
+        const currentRoute = getCurrentRoute();
+        if (currentRoute?.name !== 'Incoming Call') {
```

Comment:
Is it easy to put these route names in an enum?

Reply:
I did the changes locally and they touch _a lot_ of files, are you sure you want to include them in this PR?

Reply:
Uggh. nvm then.

Reply:
Ok, I'll shelf it for now and file a ticket.

Reply:
Here's the ticket https://issues.corp.twilio.com/browse/VBLOCKS-2844